android-kotlin/app/.gitignore

```diff
@@ -0,0 +1,2 @@
+/build
```

Comment:
Normally we try to just have the top-level .gitignore file, since that makes it easier to update.